alpine-consul/root/etc/services.d/consul-template/run

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/with-contenv sh
```

Comment:
Perhaps we could replace this shell script with an execline script that uses `s6-svwait`?